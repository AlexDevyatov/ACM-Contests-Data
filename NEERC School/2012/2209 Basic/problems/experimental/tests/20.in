249 502
428 500
455 14
273 96
303 393
395 133
231 483
33 74
117 280
169 138
311 140
407 296
105 156
114 139
423 316
185 148
351 128
398 198
210 276
126 468
361 46
211 500
423 38
207 189
186 204
129 56
332 386
49 253
216 1
271 1
17 213
302 147
8 494
83 325
255 434
272 474
358 316
217 76
166 225
338 314
248 129
55 385
313 47
72 239
38 33
33 418
142 448
454 375
107 106
445 484
98 422
68 391
203 3
121 445
375 80
248 132
160 358
64 301
7 198
138 360
123 261
297 383
174 230
309 325
320 457
261 289
412 179
143 132
429 339
72 429
233 69
494 100
326 501
253 228
167 367
446 326
394 109
246 190
240 202
6 228
90 150
62 288
156 299
289 41
323 20
227 229
358 192
72 174
300 242
365 16
392 299
428 110
310 414
59 420
231 303
168 434
77 485
446 307
65 357
101 209
243 432
79 68
279 445
211 409
196 139
405 183
245 346
131 342
196 483
218 350
290 118
118 335
30 386
393 314
475 198
67 361
39 305
339 312
160 397
137 441
132 391
164 27
355 471
451 147
13 32
381 41
499 397
342 463
447 323
89 394
208 267
134 446
85 21
22 189
3 94
26 201
284 447
265 115
395 307
302 358
331 488
146 122
113 290
150 138
245 470
38 119
277 8
439 397
443 421
397 102
462 483
164 316
41 427
378 481
122 474
341 373
295 336
199 458
265 33
105 351
216 207
415 491
273 195
3 267
383 213
272 454
147 15
414 488
58 146
245 57
192 351
35 351
303 291
403 237
19 338
172 435
347 457
316 278
331 120
264 101
71 296
385 346
263 336
13 225
40 363
120 56
79 329
81 341
216 261
351 399
264 501
192 436
21 255
471 143
453 494
51 119
52 280
154 468
378 180
460 334
185 22
4 499
383 369
101 497
66 88
28 262
127 398
258 197
406 93
91 149
193 353
456 370
408 240
225 67
52 155
330 87
207 394
304 230
20 169
155 307
247 279
50 259
420 60
140 264
76 415
235 15
496 330
185 73
458 465
250 472
76 268
422 79
97 93
27 25
246 320
486 502
187 150
456 458
496 4
445 135
253 219
142 35
430 83
186 278
7 324
9 82
81 284
247 104
339 489
386 417
0 0 1 1 0 0 2 2 0 0 0 0 1 0 1 1 1 0 0 1 1 1 0 0 0 0 1 0 0 0 0 0 4 0 1 0 0 0 1 0 3 0 0 0 0 0 1 0 1 1 1 2 0 0 1 2 0 0 0 1 0 0 0 0 0 1 1 1 1 0 0 1 0 0 0 1 0 0 3 1 1 1 1 0 0 0 1 0 0 1 0 0 2 1 0 1 0 0 0 1 1 0 0 0 1 1 0 0 0 0 0 0 1 1 0 0 1 0 1 0 1 1 1 0 0 0 1 1 0 0 1 1 1 0 1 0 1 1 0 2 0 1 0 0 0 2 2 1 1 1 0 0 0 0 0 2 0 0 0 1 0 0 0 1 0 0 0 1 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 1 1 0 0 2 0 0 3 1 0 1 1 0 0 0 0 1 0 0 1 0 0 0 0 1 0 0 0 0 0 0 2 1 0 0 0 0 0 0 0 2 0 0 2 1 1 0 0 0 0 0 0 1 0 1 1 0 0 0 0 2 2 1 2 0 0 0 0 1 0 2 0 0 1 0 0 1 1 0 1 1 0 2 0 0 0 1 1 0 0 0 1 0 1 1 0 0 0 0 2 0 0 0 1 1 1 0 0 0 0 1 1 0 0 1 1 1 0 2 1 0 0 1 0 1 0 0 0 0 2 0 1 0 0 0 0 0 0 2 0 1 2 0 0 0 0 2 0 0 1 1 1 0 1 2 0 1 1 0 0 0 0 1 0 0 1 0 0 0 0 0 0 1 2 0 1 2 0 1 0 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 3 0 1 2 0 0 0 0 1 0 1 3 1 0 3 1 1 0 0 0 0 0 1 0 1 1 1 0 0 1 0 2 1 0 1 0 0 1 1 1 2 0 0 0 0 1 2 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 2 2 0 1 0 0 1 0 0 1 1 1 1 2 0 0 0 0 0 0 1 0 0 2 0 1 2 1 0 1 1 0 0 0 0 0 0 0 3 0 1 0 0 0 0 0 1 0 0 1 0 2 1 0 1 2 0 1
