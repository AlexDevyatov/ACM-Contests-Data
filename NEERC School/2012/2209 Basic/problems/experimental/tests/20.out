500 455 96 393 133 483 33 117 169 140 407 156 114 423 148 128 398 276 468 361 500 423 189 204 56 386 49 216 271 17 147 8 325 255 474 358 217 225 314 248 55 47 239 33 33 448 454 106 445 422 68 3 121 80 248 160 301 7 360 123 383 230 309 457 289 412 132 429 429 69 100 326 228 367 326 394 246 240 228 90 288 156 41 323 229 192 72 300 16 299 428 414 420 303 168 485 446 357 209 432 79 445 409 196 405 245 131 483 350 290 335 386 314 475 361 39 339 397 137 391 164 471 451 13 41 499 342 323 394 267 446 21 189 94 201 284 265 395 358 331 146 113 138 470 119 8 397 421 397 483 316 41 378 122 341 295 458 33 105 216 491 195 267 383 272 147 414 146 245 192 35 303 237 338 435 347 278 331 101 296 385 336 225 363 56 79 81 261 399 264 192 255 471 494 51 52 468 180 334 22 4 383 497 66 262 127 258 93 149 193 456 408 67 52 87 394 304 20 307 279 50 60 140 415 15 496 185 465 472 76 79 93 27 246 502 150 458 496 135 253 142 83 186 7 82 284 247 339 417
